tests/README.md

```diff
@@ -144,15 +147,6 @@ volumes and images that might be created during the test run. The `--no-provisio
 flag is used to skip provisioning the environment, which is already done in
 step 2.
 
-With an environment provisioned separately, you could even run the tests in
```

Comment:
I would keep this, because more and more integration tests are heading in this direction and some can already be run in parallel (examples).

Reply:
If you plan on removing the `pytest-xdist` package from the development requirements, you should document here that you need it instead.

Reply:
`pytest-xdist` is already gone in #1334.

Reply:
I'll add something abuot the need to install it if you want to do that.